pkg/models/models.go

```diff
@@ -23,6 +23,7 @@ const (
 	EcosystemGo                = "Go"
 	EcosystemNpm               = "npm"
 	EcosystemPyPI              = "PyPI"
+	EcosystemUv                = "uv"
```

Comment:
No need to introduce new ecosystem. UV is a package manager for the PyPI ecosystem right?